packages/scripts/package.json

```diff
@@ -41,8 +41,11 @@
 		"eslint": "^4.19.1",
 		"jest": "^23.6.0",
 		"npm-package-json-lint": "^3.3.1",
+		"parcel-bundler": "^1.10.3",
 		"read-pkg-up": "^1.0.1",
-		"resolve-bin": "^0.4.0"
+		"resolve-bin": "^0.4.0",
+		"webpack": "4.8.3",
```

Comment:
As usual, I don't want to bump the version when doing refactoring to avoid incompatible changes.